src/LighthouseServiceProvider.php

```diff
@@ -145,6 +148,16 @@ public function provideSubscriptionResolver(FieldValue $fieldValue): Closure
             };
         });
 
+        $this->app->singleton(MiddlewareBridge::class, function (Container $app): MiddlewareBridge {
+            if ($app instanceof \Illuminate\Foundation\Application) {
+                return new LaravelMiddlewareBridge($app->get('router'));
```

Comment:
@spawnia Should we use the interface when getting from the container instead of the string shorthand?

Reply:
I'm going ahead and changing this to use the interface, but you guys should feel free to adapt the code to your own liking.